src/lambda/models/samLambdaRuntime.ts

```diff
@@ -30,6 +30,7 @@ const DEFAULT_RUNTIMES = Map<RuntimeFamily, Runtime>([
 ])
 
 export const samLambdaRuntimes: Set<Runtime> = Set.union([nodeJsRuntimes, pythonRuntimes, dotNetRuntimes])
+export const importableSamLambdaRuntimes: Set<Runtime> = Set.union([nodeJsRuntimes, pythonRuntimes])
```

Comment:
we have `samLambdaCreatableRuntimes` already, so it is more consistent to name this `samLambdaImportableRuntimes`. It also helps discoverability (and as a visual aid): these are similar concepts, all having the same prefix (`samLambdaX`)--not to mention, the same casing.